### PR TITLE
PBM-968 Updated Contributing guide with the builds of MD docs

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,22 +12,20 @@ If you'd like to submit a code patch, follow the [Contributing guide in PBM code
...
-To contribute to documentation, learn about the following:
-- [reStructured text](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) markup language. 
-- [Sphinx](https://www.sphinx-doc.org/en/master/usage/quickstart.html) documentation generator. We use it to convert source ``.rst`` files to html and PDF documents.
-- [git](https://git-scm.com/)
-- [Docker](https://docs.docker.com/get-docker/). It allows you to run Sphinx in a virtual environment instead of installing it and its dependencies on your machine.
 
 ### Edit documentation online via GitHub
 
-1. Click the **Edit this page** link on the sidebar. The source ``.rst`` file of the page opens in GitHub editor in your browser. If you haven’t worked with the repository before, GitHub creates a [fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) of it for you.
+1. Click the **Edit this page** icon next to the page title. The source `.md` file of the page opens in GitHub editor in your browser. If you haven’t worked with the repository before, GitHub creates a [fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) of it for you.
```
Comment (patrickbirch) [inline CONTRIBUTING.md:25]: pencil icon? A user may spend an hour looking for the edit this page icon.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,22 +12,20 @@ If you'd like to submit a code patch, follow the [Contributing guide in PBM code
...
 
 2. Edit the page. You can check your changes on the **Preview** tab.
 
-   **NOTE**: GitHub’s native markup language is [Markdown](https://daringfireball.net/projects/markdown/) which differs from the reStructured text. Therefore, though GitHub renders titles, headings and lists properly, some rst-specific elements like variables, directives, internal links will not be rendered.
```
Comment (patrickbirch) [inline CONTRIBUTING.md:29]: rst-specific?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -78,48 +76,98 @@ git checkout -b <my_branch>
 
 To verify how your changes look, generate the static site with the documentation. This process is called *building*. You can do it in these ways:
```
Comment (patrickbirch) [inline CONTRIBUTING.md:77]: Should you tell them which directory to use for editing?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -78,48 +76,98 @@ git checkout -b <my_branch>
...
+    ```
+
+The PDF is in `site/_pdf`.
+
+
 ## After your pull request is merged
 
 Once your pull request is merged, you are an official Percona Community Contributor. Welcome to the community!
+
+## Repository structure
```
Comment (patrickbirch) [inline CONTRIBUTING.md:148]: This information should be moved to the build locally section or at least let the user know this tree structure is available in the build locally section.
Do we need this level of detail? Would a user need to know what's in the resources tab, for example?

File: CONTRIBUTING.md (hunk 2 above)
Comment (nastena1606) [reply CONTRIBUTING.md:29]: This one is removed since PBM is in MD already

**Review by patrickbirch — APPROVED** [review]
some minor fixes

**Pull request merged**